Wikipedia/UI-V5/WMFArticlePreviewCell.m

```diff
@@ -6,12 +6,15 @@
 
 #import "NSAttributedString+WMFModify.h"
 #import "UIImageView+MWKImage.h"
+#import "NSAttributedString+WMFHTMLForSite.h"
+#import "UIFont+WMFStyle.h"
 #import "UIButton+WMFButton.h"
 #import "WMFSaveButtonController.h"
 #import "SessionSingleton.h"
+#import "NSParagraphStyle+WMFParagraphStyles.h"
 
-static CGFloat const WMFTextPadding = 8.0;
-static CGFloat const WMFImageHeight = 160;
+CGFloat const WMFArticlePreviewCellTextPadding = 8.0;
```

Comment:
They were exposed publicly, so I have to remove `static` in order for them to be `extern`.